sycl/test/basic_tests/buffer/buffer_reinterpret.cpp

```diff
@@ -0,0 +1,70 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+
+#include <sycl.hpp>
```

Comment:
```suggestion
#include <sycl/sycl.hpp>
```
I think we need to file a bug report for this. I would expect compiler to emit a error for `#include <sycl.hpp>`.

Reply:
> I think we need to file a bug report for this. I would expect compiler to emit a error for `#include <sycl.hpp>`.

I am unsure about what you are asking.
I would not expect the compiler to emit a specific error for `#include <sycl.hpp>` any further than with any `#include <whatever_not_found.hpp>`.
If someone declares at some point for an other purpose `sycl.hpp` in the include path unrelated to a SYCL impleemntation, `#include <sycl.hpp>` should work in this context, without the compiler suggesting to use `#include <sycl/sycl.hpp>` instead, right?

Reply:
I'm not asking to emit an error for `#include <sycl.hpp>` expression, but rather remove an include path, that allows users to include **standard** SYCL header with just `#include <sycl.hpp>`.